### ISSUE TITLE
Can I use Notion API instead of Ghost?

### ISSUE THREAD

**Issue by imakshayjoshi** [styxlab/next-cms-ghost#94]
Please look at this and let me know if I can use Notion API to retrieve blog posts in this script
https://developers.notion.com/

here's an example for react.js
https://alexandar.me/posts/using-notion-as-a-cms-with-react

Looking forward to a positive response.

**Comment by styxlab** [comment]
This repo has only one adapter for sourcing in Ghost CMS content. You can fork this repo and change the content source (Notion API, Wordpress, ...). I am not officially supporting it here, but it's certainly doable,

**Comment by imakshayjoshi** [comment]
ok thanks.

I also tested adding Ghost CMS content key and URL in the lib/processEnv.ts file but it is still showing your dummy content.

please help

**Comment by styxlab** [comment]
Please have a look into the [README](https://github.com/styxlab/next-cms-ghost#-ghost-content-api-keys) where I explain how to set-up the keys. You don't need to mess with files.

**Comment by imakshayjoshi** [comment]
it's done thank you.

**Comment by imakshayjoshi** [comment]
how do I make changes to the website design. Like Logo change and nav change as there is no settings option in this next.js version of ghost

**Comment by imakshayjoshi** [comment]
how to upload images?

**Comment by imakshayjoshi** [comment]
and yes I wanted to say thank you for this code as it is faster than the original ghost code.

**Comment by imakshayjoshi** [comment]
Ok thanks for responding. I found out that Vercel was throwing error whenever I change the Image domains env variable to res.cloudinary.com. Doesn't matter how I change it, from code or using Vercel's ENV variable options.

But I fixed it by deleting all the posts from original Ghost CMS and then deploy the front end on Vercel.

If I discover something useful (no-code solution) while working on the design I'll definitely let you know.

Just hope I don't use up all 500MB space on vercel

To your success 😊
Akshay
